api/v1/models/endpoint_state.go

```diff
@@ -21,16 +22,25 @@ import (
 // swagger:model EndpointState
 type EndpointState string
 
+func NewEndpointState(value EndpointState) *EndpointState {
+	return &value
+}
+
+// Pointer returns a pointer to a freshly-allocated EndpointState.
+func (m EndpointState) Pointer() *EndpointState {
+	return &m
+}
+
 const (
 
-	// EndpointStateWaitingForIdentity captures enum value "waiting-for-identity"
-	EndpointStateWaitingForIdentity EndpointState = "waiting-for-identity"
+	// EndpointStateWaitingDashForDashIdentity captures enum value "waiting-for-identity"
+	EndpointStateWaitingDashForDashIdentity EndpointState = "waiting-for-identity"
```

Comment:
This seems not desirable. Is there a way to avoid this?

Reply:
@christarazi Does not look like there is an option to get around this: https://github.com/go-swagger/go-swagger/issues/2515